app/templates/requirement/form.plush.html

```diff
@@ -1,10 +1,26 @@
 
-        <%= f.InputTag("Title") %>
-        <%= f.TextArea("Description", {rows: 5}) %>
-        <%= f.SelectTag("CreatedByUserID", {options: users, label: "User" }) %>
-        <%= f.SelectTag("RequestingDepartmentID", {options: departments, label: "Department"  }) %>
-        <%= f.SelectTag("ServiceDepartmentID", {options: departments , label: "Service Department" }) %>
-        <%= f.SelectTag("RequirementTypeID", {options: requirementTypes, label:"Requirement Type" }) %>
-        <%= f.SelectTag("RequirementSubTypeID", {options: requirementSubTypes, label: "Requirement Sub Type" }) %>
-        <button class="btn btn-success mt-3" role="submit">Save</button>
-        
\ No newline at end of file
+<div class="d-flex justify-content-between mb-4">
+        <div class="w-48">
+            <%= f.InputTag("Title") %>
+        </div>
+        <div class="w-48" >
+            <%= f.SelectTag("ServiceDepartmentID", {options: departments , label: "Service Department",class:"form-select" }) %>
+        </div>
```

Comment:
You can use "grid" to get the correct distribution for those controls. Check the Bootstrap documentation (Grid system).